package.json

```diff
@@ -7,11 +7,12 @@
     "url": "https://github.com/standardnotes/web"
   },
   "scripts": {
+    "setup": "npm run submodules && npm install",
     "start": "webpack-dev-server --progress --config webpack.dev.js",
     "watch": "webpack -w --config webpack.dev.js",
     "bundle": "webpack --config webpack.prod.js",
     "build": "bundle install && npm ci && bundle exec rails assets:precompile && npm run bundle",
-    "submodules": "git submodule update --init --force --remote",
+    "submodules": "git submodule update --init",
```

Comment:
I've found without force remote it struggles to pull changes to submodules sometimes. Might be good to just keep this if it's generally harmless.

Reply:
the problem with --remote is that it actually updates the submodule's HEAD to the repo's latest commit which in many cases is not what we want. This issue came up because the web repo is now a submodule as well and I can't just pull in any develop commit from that one.
I could keep --force in though.